src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -645,6 +650,12 @@ export default class FilingHistoryList extends Vue {
         )
       }
 
+      // add properties for Consent to Continuation Outs
+      if (this.isTypeConsentContinuationOut(filing)) {
+        item.expiry = this.apiToPacificDateTime(filing.data.consentContinuationOut?.expiry)
```

Comment:
I'm assuming that `expiry` will be an API date (eg, "2023-12-31T08:01:00.000+00:00") but maybe it will instead be "2023-12-31" and the UI will append the "12:01 AM".

This can be updated easily later when we get deeper into this new filing type.

Reply:
@vysakh-menon-aot We forgot to decide whether expiry ^^ will be a full API UTC string, or simply YYYY-MM-DD. What do you think?

What is done currently (in Lear) for corp COA filings that are effective the next day at 12:01?

Reply:
for limited restoration (filing.data.restoration) we decide to go with YYYY-MM-DD since that's how the user provides. i could do the same here

Reply:
In this case, it's not the user providing this value -- the API will calculate it (today plus 6 months, I think).

![image](https://user-images.githubusercontent.com/10002889/216734396-312971da-a1a9-4557-8782-1aa148c425c1.png)

Let's go with YYYY-MM-DD for now.